lib/dal/tests/integration_test/dependent_values_update.rs

```diff
@@ -1,509 +1,167 @@
 use dal::{
-    AttributeValue,
     Component,
     DalContext,
-    InputSocket,
-    OutputSocket,
-    Schema,
-    SchemaVariant,
     component::resource::ResourceData,
     workspace_snapshot::DependentValueRoot,
 };
 use dal_test::{
-    expected::{
-        self,
-        ExpectComponent,
-    },
+    Result,
     helpers::{
-        ChangeSetTestHelpers,
-        create_named_component_for_schema_variant_on_default_view,
+        attribute::value,
+        change_set,
+        component,
     },
     test,
 };
 use serde_json::json;
 use veritech_client::ResourceStatus;
 
+// TODO implement this! It worked for connections but does not work for subscriptions
```

Comment:
Subscriptions still work when the upstream component is ghosted; we probably want them to stop yielding a value. Adding features/fixing actual bugs is out of scope for this PR.